scripts/frontend-dev-server/index.js

```diff
@@ -73,13 +73,6 @@ const go = async () => {
         }),
     );
 
-    app.get(
-        '/static/frontend',
-        express.static(
-            path.relative(__dirname, path.resolve(root, 'frontend', 'static')),
-        ),
-    );
-
```

Comment:
Unrelated change, but this is completely unnecessary as the same middleware is applied above